README.md

```diff
@@ -203,6 +204,20 @@ can be replaced with:
 
 **) Make sure that Native Image is used only once per build job. Otherwise, the report is only generated for the last Native Image build.*
 
+## Advanced Usage
+
+### Java-version file
+
+If the `java-version-file` input is specified, the action will try to extract the version from the file and install it.
+Action is able to recognize all variants of the version description according to [jenv](https://github.com/jenv/jenv).
+Valid entry options:
+```
+major versions: 8, 11, 16, 17
```

Comment:
8, 11, 16 are rather old. Could you please update this and the other lists to 17+?